extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -524,22 +535,18 @@ export class PublishDatabaseDialog {
 		}
 	}
 
-	private createProfileRow(view: azdataType.ModelView): azdataType.FlexContainer {
-		const loadProfileButton = this.createLoadProfileButton(view);
+	private createProfileSection(view: azdataType.ModelView): azdataType.FlexContainer {
+		const selectProfileButton = this.createSelectProfileButton(view);
+		const saveProfileAsButton = this.createSaveProfileAsButton(view);
+
 		this.loadProfileTextBox = view.modelBuilder.inputBox().withProps({
 			placeHolder: constants.loadProfilePlaceholderText,
 			ariaLabel: constants.profile,
 			width: cssStyles.publishDialogTextboxWidth,
 			enabled: false
 		}).component();
 
-		const profileLabel = view.modelBuilder.text().withProps({
-			value: constants.profile,
-			width: cssStyles.publishDialogLabelWidth
-		}).component();
-
-		const profileRow = view.modelBuilder.flexContainer().withItems([profileLabel, this.loadProfileTextBox], { flex: '0 0 auto', CSSStyles: { 'margin-right': '10px' } }).withLayout({ flexFlow: 'row', alignItems: 'center' }).component();
-		profileRow.insertItem(loadProfileButton, 2, { CSSStyles: { 'margin-right': '0px' } });
+		const profileRow = view.modelBuilder.flexContainer().withItems([this.loadProfileTextBox, selectProfileButton, saveProfileAsButton], { flex: '0 0 auto', CSSStyles: { 'margin-right': '10px' } }).withLayout({ flexFlow: 'row', alignItems: 'center' }).component();
```

Comment:
can the left side of the "Select Profile" button line up with the left edge of the two input boxes below it so they're all aligned? 
![image](https://user-images.githubusercontent.com/31145923/231594260-06fcffd5-2ae1-49aa-85ab-c3794b41fecf.png)

Reply:
![image](https://user-images.githubusercontent.com/57200045/231873179-3c07ee7d-0001-40b9-922c-e25f0edcc3a7.png)
 this is the updated UI.